### PR TITLE
[case](regression-test) add backup recovery for inverted index and bl…

### PR DESCRIPTION
**Pull request by felixwluo**
…oom filter

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by w41ter — APPROVED** [review]

**Comment by w41ter** [comment]
run buildall

**Review by w41ter — APPROVED** [review]

File: regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    def syncer = getSyncer()
+    syncer.createS3Repository(repoName)
+
+    sql "CREATE DATABASE IF NOT EXISTS ${dbName}"
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableName}"
+    sql """
+        CREATE TABLE ${dbName}.${tableName} (
+            `sale_date` date NOT NULL,
+            `saler_id` int NOT NULL,
+            `category_id` int NOT NULL
```
Comment (w41ter) [inline regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy:32]: Exception:
java.sql.SQLException: errCode = 2, detailMessage = Syntax error in line 6:
        Duplicate KEY(`id`)
        ^
Encountered: DUPLICATE
Expected: COMMA

File: regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	)
```
Comment (w41ter) [inline regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy:33]:
```suggestion
        )
```

**Comment by felixwluo** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    def syncer = getSyncer()
+    syncer.createS3Repository(repoName)
+
+    sql "CREATE DATABASE IF NOT EXISTS ${dbName}"
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableName}"
+    sql """
+        CREATE TABLE ${dbName}.${tableName} (
+            `sale_date` date NOT NULL,
+            `saler_id` int NOT NULL,
```
Comment (w41ter) [inline regression-test/suites/backup_restore/test_backup_restore_bloom_filter.groovy:31]: Exception:
java.sql.SQLException: errCode = 2, detailMessage = Key columns should be a ordered prefix of the schema. KeyColumns[0] (starts from zero) is saler_id, but corresponding column is sale_date in the previous columns declaration.

**Review by w41ter — CHANGES_REQUESTED** [review]

**Comment by w41ter** [comment]
run buildall

**Comment by felixwluo** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by w41ter — APPROVED** [review]

**Pull request merged**